linux_os/guide/services/cron_and_at/package_cron_installed/rule.yml

```
@@ -8,19 +14,32 @@ rationale: 'The cron service allow periodic job execution, needed for almost all
 
 severity: medium
 
+identifiers:
+    cce@sle12: CCE-92263-3
+    cce@sle15: CCE-91379-8
+
 references:
     anssi: BP28(R50)
     cis-csc: 11,14,3,9
+    cis@sle12: 5.1.1
+    cis@sle15: 5.1.1 
     cis@ubuntu2004: 5.1.1
-    cis@ubuntu2204: 5.1.1
+    cis@ubuntu2204: 5.1.1    
```

```suggestion
    cis@ubuntu2204: 5.1.1
```
drop the extra spaces added by mistake I guess